python/cuml/cuml/accel/tests/scikit-learn/xfail-list.yaml

```diff
@@ -1318,3 +1318,6 @@
   marker: cuml-accel_kmeans_fails_with_attribute_error_memmap_ptr
   tests:
   - "sklearn.cluster.tests.test_k_means::test_predict_does_not_change_cluster_centers[None]"
+- reason: "The signs of the principal components are swapped with cuml.accel (insignificant deviation)"
+  tests:
+  - "test_equivalence_components_pca_spca[42]"
```

Comment:
Looks like this test is still running [on CI]( https://github.com/rapidsai/cuml/actions/runs/14917987050/job/41908356516?pr=6648#step:9:46067 )

Do we need to prefix the module like with the other tests?

```suggestion
  - "sklearn.decomposition.tests.test_sparse_pca.py::test_equivalence_components_pca_spca[42]"
```

Or is something else needed?

Reply:
Yes, just realized that mistake. Pushing a fix now.